app/players/player.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname

```

Comment:
I am not sure that this method is needed as every class has the `__init__` method